CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration_UI.uc

```diff
@@ -511,5 +512,39 @@ static protected function EventListenerReturn StrategyPolicy_ShowCovertActionsOn
 	// Never show actions popup after UIStrategyPolicy
 	Tuple.Data[0].b = false;
 	
+	return ELR_NoInterrupt;
+}
+
+////////////////////
+/// Tactical HUD ///
+////////////////////
+
+static function CHEventListenerTemplate CreateTacticalHUDListeners()
+{
+	local CHEventListenerTemplate Template;
+
+	`CREATE_X2TEMPLATE(class'CHEventListenerTemplate', Template, 'Infiltration_UI_TacticalHUD');
+	Template.AddCHEvent('OverrideReinforcementsAlert', IncomingReinforcementsDisplay, ELD_Immediate);
+	Template.RegisterInTactical = true;
+
+	return Template;
+}
+
+static function EventListenerReturn IncomingReinforcementsDisplay(Object EventData, Object EventSource, XComGameState GameState, Name EventID, Object CallbackData)
+{
+	local XComLWTuple Tuple;
+
+	Tuple = XComLWTuple(EventData);
+
+	if (Tuple == none)
+	{
+		return ELR_NoInterrupt;
+	}
+
+	if (class'XComGameState_CIReinforcementsManager'.static.SetCountdownDisplay(Tuple))
```

Comment:
Check the vanilla AI rnf spawner as well - so that if it has `countdown > 1` then the altered display is used as well

Reply:
the manager already checks it's inside the update function

Reply:
Ugh, this is really confusing. The only thing that the manager should be doing is

1) getting "orders" to spawn rnfs in x turns
2) spawning rnfs when needed

The UI logic should be inside the listener, and the vanilla AI rnf spawner should not be "rolled into" our manager - they way you have it now is really confusing